indico/modules/core/captcha.py

```diff
@@ -24,11 +24,12 @@
 
 def _verify_captcha(user_answer):
     if plugin := get_captcha_plugin():
-        return plugin.validate_captcha(user_answer)
-    else:
-        answer = session.get('captcha_state')
-        # 1 and 7 are too similar and we never use 1 in a challenge
-        return bool(answer) and answer == user_answer.replace('1', '7')
+        captcha_validation = plugin.validate_captcha(user_answer)
```

Comment:
you never return the value... so right now plugin captchas are most likely broken w/ your PR

(PS: hcaptcha is easy to set up, you can easily test this yourself w/ our cloud-captchas plugin)